standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -649,6 +649,10 @@ public enum ConfVars {
     COMPACTOR_CLEANER_TABLECACHE_ON("metastore.compactor.cleaner.tablecache.on",
             "hive.compactor.cleaner.tablecache.on", true,
             "Enable table caching in the cleaner. Currently the cache is cleaned after each cycle."),
+    COMPACTOR_CLEAN_ABORTS_USING_CLEANER("metastore.compactor.clean.aborts.using.cleaner", "hive.compactor.clean.aborts.using.cleaner", true,
```

Comment:
no need for extra config

Reply:
The plan is to keep this config till some point until we know that abort cleanup is stable. If there are any issues with the given handler, we can use the compaction cycle. I will create a task once its determined that this feature is stable and we can remove this feature flag and associated logic as well.